pkg/alicloud/errors/codes.go

```diff
@@ -27,6 +27,8 @@ const (
 	InvalidVpcZoneNotSupported = "InvalidVpcZone.NotSupported"
 	// InvalidZoneIDNotSupportShareEncryptedImage: Creating instances by shared encrypted images is not supported in this zone.
 	InvalidZoneIDNotSupportShareEncryptedImage = "InvalidZoneId.NotSupportShareEncryptedImage"
+	// InvalidResourceType.NotSupported: The specified resource does not exist. Please try other zone or configuration.
```

Comment:
The godoc convention is that they start with the name of the variable/function
```suggestion
	// InvalidResourceTypeNotSupported: The specified resource does not exist. Please try other zone or configuration.
```